drivers/SmartThings/matter-switch/src/aqara-cube/init.lua

```diff
@@ -0,0 +1,216 @@
+local capabilities = require "st.capabilities"
+local clusters = require "st.matter.generated.zap_clusters"
+local device_lib = require "st.device"
+
+local cubeAction = capabilities["stse.cubeAction"]
+local cubeFace = capabilities["stse.cubeFace"]
+
+local COMPONENT_TO_ENDPOINT_MAP_BUTTON = "__component_to_endpoint_map_button"
+local DEFERRED_CONFIGURE = "__DEFERRED_CONFIGURE"
+local INITIAL_PRESS_ONLY = "__initial_press_only" -- for devices that support MS (MomentarySwitch), but not MSR (MomentarySwitchRelease)
+
+local CUBEACTION_TIMER = "cubeAction_timer"
```

Comment:
```suggestion
local CUBEACTION_TIMER = "__cubeAction_timer"
```
This is generally our admittedly arbitrary syntax

Reply:
I changed it.